internal/core/platform.rs

```diff
@@ -16,7 +16,12 @@ use crate::unsafe_single_core::{thread_local, OnceCell};
 #[cfg(feature = "std")]
 use once_cell::sync::OnceCell;
 
-use crate::window::PlatformWindow;
+pub use crate::items::{InputType, MouseCursor};
+pub use crate::lengths::{PhysicalLength, PhysicalPoint};
+pub use crate::renderer::Renderer;
+#[cfg(feature = "swrenderer")]
+pub use crate::swrenderer;
+pub use crate::window::PlatformWindow;
```

Comment:
I suppose that in a separate step we could move `PlatformWindow` into this module directly?